src/common.cmake

```diff
@@ -21,13 +21,6 @@ elseif(${CMAKE_SYSTEM_NAME} STREQUAL "Darwin")
 endif()
 
 
-### build type
-
-if(NOT CMAKE_BUILD_TYPE)
```

Comment:
Fine. Would still be interested to know why it wasn't picked up.

Reply:
did not grok this fully, just saw by experiment that CMAKE_BUILD_TYPE had the value "Debug" which explained the MSI bug... but officially, the default value seems to be undefined (maybe even depending on the Generator). -> best practice is to always set it from the outside